### ISSUE TITLE
unescaped single quote in username have browser display blank screen.

### ISSUE THREAD

**Issue by schplurtz** [Ylianst/MeshCentral#1324]
Hi,

meshcentral is impressive. thanks a lot.

I was hit by a tiny bug today. When user friendly name contains a singe quote, user can log in but gets a blank screen.

meshcentral version : 0.5.27
node version : 14.2.0

To reproduce, use this config.json snippet and login as anne.
```javascript
  "domains": {
    "": {
    "newAccounts": 0,
    "Auth": "ldap",
    "ldapusername": "gecos",
    "ldapuserkey": "uid",
    "ldapoptions": {
      "url": "test",
      "anne": {
        "gecos": "Anne O'Nyme",
        "uid": "anneonyme"
      },
      "so": {
        "gecos": "Sophie Sticker",
        "uid": "ssticker"
      }
    }
```
User is created in mongodb. I can check with mongo shell
```
db.meshcentral.find({"_id": "user//anneonyme"}) 
{ "_id" : "user//anneonyme", "type" : "user", "name" : "Anne O'Nyme", "creation" : 1589047184, "login" : 1589047259, "domain" : "", "pastlogin" : 1589047184 }
```

Browser console indicates : 
```
SyntaxError: missing ) after argument list
```
Faulty line is 1216:88
```javascript
        var logoutControls = JSON.parse(decodeURIComponent('%7B%22name%22%3A%22Anne%20O'Nyme%22%2C%22logoutUrl%22%3A%22%2Flogout%3F0.47474808924812706%22%7D'));
```

I was able to track this down to `webserver.js` and deploy a Q&D fix. I simply added `.replace(/'/g,'%27')` to each `logoutControls` value  in `render` calls.  I probably missed some places where a fix must be applied.
```diff
- render(...., logoutControls: encodeURIComponent(JSON.stringify(logoutcontrols)), ...... );
+ render(...., logoutControls: encodeURIComponent(JSON.stringify(logoutcontrols)).replace(/'/g,'%27'), ...... );
```

regards

**Comment by Ylianst** [comment]
Oh wow! Thank you for the report, that is exactly the correct solution. Just published MeshCentral v0.5.28 with this fix and updated all my servers right away. I did find a few other places that where impacted by this and fixed them. Thanks! Let me know if v0.5.28 works for you.

I also fixed the "Online" filter in this version so that devices that disconnect/connect show up correctly.

**Comment by schplurtz** [comment]
Already fixed !? Much appreciated.

0.5.28 works perfectly.
thanks again.
